shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -362,6 +362,10 @@ static bool DidFlagChange(const flutter::SemanticsNode& oldNode,
     NSString* message = annotatedEvent[@"data"][@"message"];
     ios_delegate_->PostAccessibilityNotification(UIAccessibilityAnnouncementNotification, message);
   }
+  if ([type isEqualToString:@"focus"]) {
+    SemanticsObject* node = objects_.get()[annotatedEvent[@"nodeId"]];
+    ios_delegate_->PostAccessibilityNotification(UIAccessibilityLayoutChangedNotification, node);
```

Comment:
What happens if `node` is nil? Seems like nil is an OK arg, but just double checking

Reply:
Tested in my device, if node is nil, nothing will happen, the focus will not be changed